src/components/tabController/useScrollToItem.ts

```diff
@@ -177,12 +178,13 @@ const useScrollToItem = <T extends ScrollToSupportedViews>(props: ScrollToItemPr
 
   const focusIndex = useCallback((index: number, animated = true) => {
     if (index >= 0 && offsets.CENTER.length > index) {
+      const rtlIndex = Constants.isRTL && Constants.isIOS ? itemsCount - index - 1 : index;
```

Comment:
Let's move `Constants.isRTL && Constants.isIOS` to a const at the top of the file like we did before so it's easier to spot there's an RTL fix